server/test/categoryRouter.test.js

```diff
@@ -82,7 +115,6 @@ describe('categoryRouter', () => {
         })
         .set('content-type', 'application/json')
         .end(async function (error, response) {
-          expect(error).to.be.null;
```

Comment:
Why was this line removed?

Reply:
Because we never send error bc it's always reflected in the response status, so i figured we wouldn't need

Reply:
sgtm